recipe/build.sh

```diff
@@ -18,6 +18,10 @@ if [[ ${cuda_compiler_version} != "None" && "$target_platform" == linux-64 ]]; t
         export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0;7.5;8.0;8.6+PTX"
```

Comment:
Generally I'd prefer to remove settings for obsolete CUDA versions from `build.sh` wherever we migrate CUDA. But that can be done in a follow-up.